src/include/duckdb/function/table/table_scan.hpp

```diff
@@ -28,6 +28,8 @@ struct TableScanBindData : public TableFunctionData {
 	bool is_index_scan;
 	//! Whether or not the table scan is for index creation.
 	bool is_create_index;
+	//! In what order to scan the row groups
+	shared_ptr<RowGroupOrderOptions> order_options;
```

Comment:
Unless I'm missing something this doesn't seem to ever be shared - given that it's not shared can this be a `unique_ptr`?